### PR TITLE
Implement batch explorer api

### PR DESCRIPTION
**Pull request by ercecan**
# Description
Implements batch explorer api

### CODE REVIEW

File: batch-explorer/README.md
Comment (eyusufatik) [inline batch-explorer/README.md:1]: adding a small readme explaining why we have it here would be nice

File: batch-explorer/app.py [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+app = FastAPI(lifespan=lifespan)
+
+app.add_middleware(TrustedHostMiddleware, allowed_hosts=["*"])
+
+
+origins = [
+    "*",
+]
+
```
Comment (eyusufatik) [inline batch-explorer/app.py:39]: should be settable by config

File: batch-explorer/services/l2_rpc.py
Comment (eyusufatik) [inline batch-explorer/services/l2_rpc.py:1]: can use ethers-py here

File: batch-explorer/dtos/commitment.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
+from pydantic import BaseModel
+
+
+class CommitmentResponse(BaseModel):
+    l1_tx_id: str
+    l1_start_hash_mempool_url: str
+    l1_end_hash_mempool_url: str
+    l2_start_height_block_exp_url: str
+    l2_end_height_block_exp_url: str
+    merkle_root: str
```
Comment (eyusufatik) [inline batch-explorer/dtos/commitment.py:10]: return heights and hashes as well along with urls to explorers

**Review by eyusufatik — CHANGES_REQUESTED** [review]

File: batch-explorer/services/l2_rpc.py
Comment (ercecan) [reply batch-explorer/services/l2_rpc.py:1]: that is not very well maintained I believe,
I can use web3py?

**Pull request merged**